features/cellular/framework/device/CellularStateMachine.h

```diff
@@ -177,9 +181,10 @@ class CellularStateMachine {
     bool _command_success;
     bool _is_retry;
     cell_callback_data_t _cb_data;
-    nsapi_event_t _current_event;
+    cellular_connection_status_t _current_event;
     int _status;
     PlatformMutex _mutex;
+    cell_signal_quality _signal_quality;
```

Comment:
`cell_signal_quality ` vs `cellular_connection_status_t ` - not having the same name scheme

Reply:
fixed